nodejs/src/v5/analytics.test.js

```diff
@@ -113,7 +113,7 @@ ad_group_ids=test_ad_group\
 &engagement_window_days=14\
 &granularity=HOUR');
 
-    analytics.view_window_days(60).conversion_report_time('AD_EVENT');
+    analytics.view_window_days(60).conversion_report_time('TIME_OF_CONVERSION');
```

Comment:
The equivalent enum value to `AD_EVENT` would be `TIME_OF_AD_ACTION`

Reply:
thanks for the comment! i understand. that's a unit test, and provides no functionality besides ensuring that the module is willing to except that value for the parameter. i made the change to ensure that the particular problem reported by a customer would be addressed.